library/core/class.oauth2.php

```diff
@@ -400,7 +401,9 @@ public function settingsEndpoint($sender, $args) {
             'AuthorizeUrl' =>  ['LabelCode' => 'Authorize Url', 'Description' => 'URL where users sign-in with the authentication provider.'],
             'TokenUrl' => ['LabelCode' => 'Token Url', 'Description' => 'Endpoint to retrieve the authorization token for a user.'],
             'ProfileUrl' => ['LabelCode' => 'Profile Url', 'Description' => 'Endpoint to retrieve a user\'s profile.'],
-            'BearerToken' => ['LabelCode' => 'Authorization Code in Header', 'Description' => 'When requesting the profile, pass the access token in the HTTP header. i.e Authorization: Bearer [accesstoken]', 'Control' => 'checkbox']
+            'BearerToken' => ['LabelCode' => 'Authorization Code in Header', 'Description' => 'When requesting the profile, pass the access token in the HTTP header. i.e Authorization: Bearer [accesstoken]', 'Control' => 'checkbox'],
+            'Prompt' => ['LabelCode' => 'Prompt', 'Description' => 'Prompt Parameter to append to Authorize Url', 'Control' => 'DropDown', 'Items' => [ 'none' => 'none', 'consent' => 'consent', 'login' => 'login', 'consent and login' =>  'consent and login']],
+            'Response Type' => ['LabelCode' => 'Code Token', 'Description' => 'The response type send in the authorize uri (note: certain values can change the work flow ie. id_token)'],
```

Comment:
Here I think the wording could be more descriptive.

The LabelCode should be "Response Type". The description could be something like:

 > `response_type` to be sent in the Authorization Request.

Also, can we add 'code token' to the defaults in the structure() function when you turn on the plugin.